06.bowling_object/frame.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class Frame
+  attr_reader :frame_number, :shot_scores
+
+  def initialize(frame_number)
+    @frame_number = frame_number
+    @shot_scores = []
+    @bonus_scores = []
+  end
+
+  def record_shot(shot_score)
+    @bonus_scores << shot_score if (strike? && @bonus_scores.size < 2 || spare? && @bonus_scores.empty?) && frame_number < 9
+    @shot_scores << shot_score if !finished?
```

Comment:
```suggestion
    @shot_scores << shot_score unless finished?
```

好みの問題ですが、unlessを使うのもありかなーと思います

Reply:
否定演算子を使うかunlessを使うかはいつも判断が難しいです

unlessを使うように修正しました！